src/api/ITooltip.js

```diff
@@ -60,6 +61,13 @@
         this.tooltip = d3Tip()
             .attr("class", "d3-tip")
             .offset(function (d) {
+                if(event && context.tooltipFollowMouse()){
+                    var d3tipElement = document.querySelector(".d3-tip"); // d3Tip offers no reference to the '.d3-tip' element...?
+                    d3tipElement.style.display = "block";
+                    d3tipElement.style.left = context.tooltipOffset() + (event.clientX) + "px";
+                    d3tipElement.style.top = event.clientY + "px";
+                    return [];
+                }
```

Comment:
Is it ok to use event this way (I am guessing so)?

Reply:
It seems okay to me.